src/Controller/DefaultController.php

```diff
@@ -448,10 +472,9 @@ public function islandora_content_model_autocomplete(Request $request) {
   /**
    * Autocomplete the MIME type name.
    */
-  public function islandora_mime_type_autocomplete(Request $request) {
-    require_once \Drupal::root() . "/includes/file.mimetypes.inc";
+  public function islandoraMimeTypeAutocomplete(Request $request) {
     $string = $request->query->get('q');
-    $mime_types = file_mimetype_mapping();
+    $mime_types = $this->mimeTypeMapper->getMapping();
```

Comment:
This method [doesn't appear to exist](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21File%21MimeType%21ExtensionMimeTypeGuesser.php/class/ExtensionMimeTypeGuesser/8.5.x)...